server/actions/settings.js

```diff
@@ -0,0 +1,31 @@
+import dbConnect from "../mongodb/index";
+const AppSettings = require("../mongodb/models/settings");
+
+export const inviteAdmin = async (email) => {
+    try {
+        await dbConnect();
+        AppSettings.invitedAdmins.push(email);
```

Comment:
You need to add a query here to find the mongo document and then update it.  You can't just access the AppSettings mongo model
Example:  check the server/actions/events.js updateEvent(updateEventData, next) on line 70.
It will be something like `AppSettings.findOneAndUpdate({}, 'update here')`
Definitely look up something along the lines of how to modify an array in mongo